models/quix/optimism/quix_v4_optimism_events.sql

```diff
@@ -0,0 +1,260 @@
+    {% endif %}
+    {% if is_incremental() %}
+    and tx.block_time >= date_trunc("day", now() - interval '1 week')
+    {% endif %}
+left join {{ ref('nft_aggregators') }} as agg
+    on agg.contract_address = tx.to 
+    and agg.blockchain = 'optimism'
+left join {{ ref('tokens_nft') }} n
+    on n.contract_address = er.nft_contract_address 
+    and n.blockchain = 'optimism'
+left join {{ source('erc721_optimism','evt_transfer') }} as erct2 
+    on erct2.evt_block_time=er.block_time
+    and er.nft_contract_address=erct2.contract_address
+    and erct2.evt_tx_hash=er.tx_hash
+    and erct2.tokenId=er.token_id
+    and erct2.to=er.buyer
+    {% if not is_incremental() %}
+    -- smallest block number for source tables above
+    and erct2.evt_block_number >= '{{min_block_number}}'
+    {% endif %}
+    {% if is_incremental() %}
+    and erct2.evt_block_time >= date_trunc("day", now() - interval '1 week')
+    {% endif %}
+left join {{ source('erc1155_optimism','evt_transfersingle') }} as erc1155 
+    on erc1155.evt_block_time=er.block_time
+    and er.nft_contract_address=erc1155.contract_address
+    and erc1155.evt_tx_hash=er.tx_hash
+    and erc1155.id=er.token_id
+    and erc1155.to=er.buyer
+    {% if not is_incremental() %}
+    -- smallest block number for source tables above
+    and erc1155.evt_block_number >= '{{min_block_number}}'
+    {% endif %}
+    {% if is_incremental() %}
+    and erc1155.evt_block_time >= date_trunc("day", now() - interval '1 week')
+    {% endif %}
+left join {{ source('erc20_optimism','evt_transfer') }} as erc20 
+    on erc20.evt_block_time=er.block_time
+    and erc20.evt_tx_hash=er.tx_hash
+    and erc20.to=er.seller
+    {% if not is_incremental() %}
+    -- smallest block number for source tables above
+    and erc20.evt_block_number >= '{{min_block_number}}'
+    {% endif %}
+    {% if is_incremental() %}
+    and erc20.evt_block_time >= date_trunc("day", now() - interval '1 week')
+    {% endif %}
+left join {{ ref('tokens_erc20') }} as t1
+    on t1.contract_address =
+        case when (erc20.contract_address = '0x0000000000000000000000000000000000000000' or erc20.contract_address is null)
+        then '0xdeaddeaddeaddeaddeaddeaddeaddeaddead0000'
+        else erc20.contract_address
+        end 
+    and t1.blockchain = 'optimism'
+    left join {{ source('prices', 'usd') }} as p1
+    on p1.contract_address =
+        case when (erc20.contract_address = '0x0000000000000000000000000000000000000000' or erc20.contract_address is null)
+        then '0xdeaddeaddeaddeaddeaddeaddeaddeaddead0000'
+        else erc20.contract_address
+        end
+    and p1.minute = date_trunc('minute', er.block_time)
+    and p1.blockchain = 'optimism'
+    {% if is_incremental() %}
+    and p1.minute >= date_trunc("day", now() - interval '1 week')
+    {% endif %}
+left join transfers as tr 
+    on tr.tx_hash = er.tx_hash 
+    and tr.block_number = er.block_number
+{{ dbt_utils.group_by(n=37) }}
```

Comment:
This is the time `group by` macro is used in the project.
And this line sould be compiled into 
`group by 1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16,17,18,19,20,21,22,23,24,25,26,27,28,29,30,31,32,33,34,35,36,37`
That would be quite a lot of keys to group by.
I will test if I remove this line the dbt-test will run as normal.